backend/repository/data/handler/test_cases.json

```diff
@@ -2700,6 +2764,18 @@
 							"thriftFile": "clients/googlenow/googlenow.thrift",
 							"type": "http"
 						},
+						"multi": {
+							"exposedMethods": {
+								"HelloA": "ServiceABack::hello",
```

Comment:
we'll have to fix how `HelloA` is generated in here: https://github.com/uber/zanzibar/blob/master/backend/repository/client_update.go#L89

I believe now it still use method name as key, that will be overriden